pandas/tests/indexes/period/test_indexing.py

```diff
@@ -206,38 +203,42 @@ def test_getitem_seconds(self):
             for d in ["2013/01/01", "2013/01", "2013"]:
                 tm.assert_series_equal(ser[d], ser)
 
-    def test_getitem_day(self):
+    @pytest.mark.parametrize(
+        "idx",
+        [
+            date_range(start="2013/01/01", freq="D", periods=400),
+            period_range(start="2013/01/01", freq="D", periods=400),
```

Comment:
can you parametrize over the functions [date_range, period_range] to avoid materializing these at test-collection time